From: Aleix Pol <aleixpol@kde.org>
Date: Mon, 11 Jan 2016 15:41:20 +0000
Subject: Rename muon-discover to plasma-discover
X-Git-Url: http://quickgit.kde.org/?p=discover.git&a=commitdiff&h=5599c1dd7b992d7ecf0279fce335168a683b8b10
---
Rename muon-discover to plasma-discover

Changes the name of some .po file names, CC'ing l10n.

REVIEW: 126680
CCMAIL: "Localization and Translation (l10n)" <kde-i18n-doc@kde.org>
---


--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,7 +4,7 @@
 
 cmake_minimum_required(VERSION 2.8.12)
 
-option(INSTALL_EXPORTER "Install muon-exporter [development utility]" OFF)
+option(INSTALL_EXPORTER "Install plasma-discover-exporter [development utility]" OFF)
 
 find_package(ECM REQUIRED NO_MODULE)
 

--- a/Messages.sh
+++ b/Messages.sh
@@ -3,10 +3,10 @@
 $EXTRACTRC `find updater/ -name \*.rc -o -name \*.ui -o -name \*.kcfg` >> updaterrc.cpp
 $EXTRACTRC --context="Category" --tag-group=none --tag=Name `find libdiscover -name "*-categories.xml"` >> categoriesxml.cpp
 $XGETTEXT categoriesxml.cpp `find libdiscover -name \*.cpp` -o $podir/libdiscover.pot
-$XGETTEXT rc.cpp updaterrc.cpp `find updater -name \*.cpp` -o $podir/muon-updater.pot
-$XGETTEXT `find discover -name \*.cpp -o -name \*.qml -o -name \*.js` -o $podir/muon-discover.pot
-$XGETTEXT `find exporter -name \*.cpp` -o $podir/muon-exporter.pot
-$XGETTEXT `find notifier -name \*.cpp` -o $podir/muon-notifier.pot
+$XGETTEXT rc.cpp updaterrc.cpp `find updater -name \*.cpp` -o $podir/plasma-discover-updater.pot
+$XGETTEXT `find discover -name \*.cpp -o -name \*.qml -o -name \*.js` -o $podir/plasma-discover.pot
+$XGETTEXT `find exporter -name \*.cpp` -o $podir/plasma-discover-exporter.pot
+$XGETTEXT `find notifier -name \*.cpp` -o $podir/plasma-discover-notifier.pot
 rm -f muonrc.cpp
 rm -f updaterrc.cpp
 rm -f categoriesxml.cpp

--- a/discover/CMakeLists.txt
+++ b/discover/CMakeLists.txt
@@ -3,7 +3,7 @@
 
 include_directories(${CMAKE_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}/..)
 
-set(muon_discover_SRCS
+set(plasma_discover_SRCS
     main.cpp
     MuonDiscoverMainWindow.cpp
 
@@ -12,11 +12,11 @@
     IconColors.cpp
 )
 
-qt5_add_resources(muon_discover_SRCS resources.qrc)
+qt5_add_resources(plasma_discover_SRCS resources.qrc)
 
-add_executable(muon-discover ${muon_discover_SRCS})
+add_executable(plasma-discover ${plasma_discover_SRCS})
 
-target_link_libraries(muon-discover LINK_PUBLIC KF5::KIOWidgets
+target_link_libraries(plasma-discover LINK_PUBLIC KF5::KIOWidgets
                                      KF5::Declarative
                                      KF5::DBusAddons
                                      KF5::I18n
@@ -26,9 +26,9 @@
                                      Discover::Common
 )
 
-install(TARGETS muon-discover ${INSTALL_TARGETS_DEFAULT_ARGS} )
+install(TARGETS plasma-discover ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
 install(PROGRAMS org.kde.discover.desktop DESTINATION ${XDG_APPS_INSTALL_DIR} )
-install(FILES featured.json DESTINATION ${DATA_INSTALL_DIR}/muondiscover)
-install(FILES muondiscoverui.rc  DESTINATION  ${KXMLGUI_INSTALL_DIR}/muondiscover)
+install(FILES featured.json DESTINATION ${DATA_INSTALL_DIR}/plasmadiscover)
+install(FILES plasmadiscoverui.rc  DESTINATION  ${KXMLGUI_INSTALL_DIR}/plasmadiscover)
 

--- a/discover/icons/CMakeLists.txt
+++ b/discover/icons/CMakeLists.txt
@@ -1,11 +1,11 @@
 ecm_install_icons(
     ICONS
-        128-apps-muondiscover.png
-        16-apps-muondiscover.png
-        22-apps-muondiscover.png
-        32-apps-muondiscover.png
-        48-apps-muondiscover.png
-        sc-apps-muondiscover.svgz
+        128-apps-plasmadiscover.png
+        16-apps-plasmadiscover.png
+        22-apps-plasmadiscover.png
+        32-apps-plasmadiscover.png
+        48-apps-plasmadiscover.png
+        sc-apps-plasmadiscover.svgz
     DESTINATION ${ICON_INSTALL_DIR}
     THEME hicolor
 )

--- a/discover/main.cpp
+++ b/discover/main.cpp
@@ -44,7 +44,7 @@
     app.setWindowIcon(QIcon::fromTheme(QStringLiteral("muondiscover")));
     app.setAttribute(Qt::AA_DontCreateNativeWidgetSiblings);
     app.setAttribute(Qt::AA_UseHighDpiPixmaps, true);
-    KLocalizedString::setApplicationDomain("muon-discover");
+    KLocalizedString::setApplicationDomain("plasma-discover");
     KAboutData about(QStringLiteral("muondiscover"), i18n("Discover"), version, i18n("An application explorer"),
                      KAboutLicense::GPL, i18n("© 2010-2014 Plasma Development Team"));
     about.addAuthor(i18n("Aleix Pol Gonzalez"), QString(), QStringLiteral("aleixpol@blue-systems.com"));

--- a/discover/muondiscoverui.rc
+++ /dev/null
@@ -1,11 +1,1 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<gui name="muon-discover"
-     version="5"
-     xmlns="http://www.kde.org/standards/kxmlgui/1.0"
-     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-     xsi:schemaLocation="http://www.kde.org/standards/kxmlgui/1.0
-                         http://www.kde.org/standards/kxmlgui/1.0/kxmlgui.xsd">
-    <ToolBar name="mainToolBar" deleted="true"/>
-    <ToolBar name="discoverToolBar" deleted="true"/>
-</gui>
 

--- a/discover/org.kde.discover.desktop
+++ b/discover/org.kde.discover.desktop
@@ -24,7 +24,7 @@
 Name[x-test]=xxDiscoverxx
 Name[zh_CN]=发现者
 MimeType=x-scheme-handler/appstream
-Exec=muon-discover %U
+Exec=plasma-discover %U
 Icon=muondiscover
 Type=Application
 X-DocPath=muon/index.html

--- /dev/null
+++ b/discover/plasmadiscoverui.rc
@@ -1,1 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<gui name="plasma-discover"
+     version="5"
+     xmlns="http://www.kde.org/standards/kxmlgui/1.0"
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://www.kde.org/standards/kxmlgui/1.0
+                         http://www.kde.org/standards/kxmlgui/1.0/kxmlgui.xsd">
+    <ToolBar name="mainToolBar" deleted="true"/>
+    <ToolBar name="discoverToolBar" deleted="true"/>
+</gui>
 

--- a/exporter/CMakeLists.txt
+++ b/exporter/CMakeLists.txt
@@ -1,10 +1,10 @@
 include_directories(${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR})
 
-add_executable(muon-exporter main.cpp MuonExporter.cpp)
+add_executable(plasma-discover-exporter main.cpp MuonExporter.cpp)
 
-target_link_libraries(muon-exporter Discover::Common KF5::CoreAddons KF5::I18n)
+target_link_libraries(plasma-discover-exporter Discover::Common KF5::CoreAddons KF5::I18n)
 
 if(MUON_INSTALL_EXPORTER)
-    install(TARGETS muon-exporter ${INSTALL_TARGETS_DEFAULT_ARGS})
+    install(TARGETS plasma-discover-exporter ${INSTALL_TARGETS_DEFAULT_ARGS})
 endif()
 

--- a/exporter/main.cpp
+++ b/exporter/main.cpp
@@ -31,7 +31,7 @@
 int main(int argc, char** argv)
 {
     QApplication app(argc, argv);
-    KLocalizedString::setApplicationDomain("muon-exporter");
+    KLocalizedString::setApplicationDomain("plasma-discover-exporter");
     KAboutData about(QStringLiteral("muonexporter"), i18n("Muon Exporter"), version, i18n(description),
                      KAboutLicense::GPL, i18n("©2013 Aleix Pol Gonzalez"), QString());
     about.addAuthor(i18n("Jonathan Thomas"), QString(), QStringLiteral("echidnaman@kubuntu.org"));

--- a/libdiscover/CMakeLists.txt
+++ b/libdiscover/CMakeLists.txt
@@ -59,5 +59,5 @@
 target_include_directories(DiscoverCommon PRIVATE ${PHONON_INCLUDES} PUBLIC ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
 
 install(TARGETS DiscoverCommon ${INSTALL_TARGETS_DEFAULT_ARGS})
-install(FILES resources/muonabstractnotifier.notifyrc DESTINATION ${KNOTIFYRC_INSTALL_DIR})
+install(FILES resources/discoverabstractnotifier.notifyrc DESTINATION ${KNOTIFYRC_INSTALL_DIR})
 

--- a/libdiscover/backends/ApplicationBackend/Application.cpp
+++ b/libdiscover/backends/ApplicationBackend/Application.cpp
@@ -495,7 +495,7 @@
         return Installed;
     }
     
-    return None; // Actually: none of interest to us here in muon-discover.
+    return None; // Actually: none of interest to us here in plasma-discover.
 }
 
 void Application::fetchScreenshots()

--- a/libdiscover/backends/PackageKitBackend/CMakeLists.txt
+++ b/libdiscover/backends/PackageKitBackend/CMakeLists.txt
@@ -15,8 +15,8 @@
 install(FILES packagekit-backend-categories.xml DESTINATION ${DATA_INSTALL_DIR}/libdiscover/categories)
 
 #notifier
-add_library(MuonPackageKitNotifier MODULE PackageKitNotifier.cpp)
-target_link_libraries(MuonPackageKitNotifier PRIVATE PK::packagekitqt5 Discover::Notifiers)
+add_library(DiscoverPackageKitNotifier MODULE PackageKitNotifier.cpp)
+target_link_libraries(DiscoverPackageKitNotifier PRIVATE PK::packagekitqt5 Discover::Notifiers)
 
-install(TARGETS MuonPackageKitNotifier DESTINATION ${PLUGIN_INSTALL_DIR}/discover-notifier)
+install(TARGETS DiscoverPackageKitNotifier DESTINATION ${PLUGIN_INSTALL_DIR}/discover-notifier)
 

--- a/libdiscover/backends/PackageKitBackend/PackageKitUpdater.h
+++ b/libdiscover/backends/PackageKitBackend/PackageKitUpdater.h
@@ -52,7 +52,7 @@
         virtual QString statusDetail() const override;
         virtual quint64 downloadSpeed() const override;
 
-        /** in muon-updater, actions with HighPriority will be shown in a KMessageWidget,
+        /** in plasma-discover-updater, actions with HighPriority will be shown in a KMessageWidget,
          *  normal priority will go right on top of the more menu, low priority will go
          *  to the advanced menu
          */

--- a/libdiscover/resources/AbstractResourcesBackend.h
+++ b/libdiscover/resources/AbstractResourcesBackend.h
@@ -133,7 +133,7 @@
         /**
          *  This method is used to integrate advanced functions into the Muon GUI.
          *
-         *  In muon-updater, actions with HighPriority will be shown in a KMessageWidget,
+         *  In plasma-discover-updater, actions with HighPriority will be shown in a KMessageWidget,
          *  normal priority will go right on top of the more menu, low priority will go
          *  to the advanced menu.
          */

--- /dev/null
+++ b/libdiscover/resources/discoverabstractnotifier.notifyrc
@@ -1,1 +1,66 @@
+[Global]
+IconName=applications-system
+Comment=Discover
 
+[Event/Update]
+Name=Updates Are Available
+Name[ca]=Hi ha actualitzacions disponibles
+Name[ca@valencia]=Hi ha actualitzacions disponibles
+Name[da]=Opdateringer tilgængelige
+Name[de]=Es sind Aktualisierungen verfügbar
+Name[el]=Υπάρχουν διαθέσιμες ενημερώσεις
+Name[en_GB]=Updates Are Available
+Name[es]=Existen actualizaciones disponibles
+Name[fi]=Päivityksiä on saatavilla
+Name[fr]=Des mises à jour sont disponibles
+Name[gl]=Hai actualizacións dispoñíbeis
+Name[it]=Sono disponibili aggiornamenti
+Name[ko]=업데이트 사용 가능
+Name[nb]=Oppgraderinger er tilgjengelige
+Name[nl]=Bijwerken is beschikbaar
+Name[nn]=Oppdateringar er tilgjengelege
+Name[pl]=Uaktualnienia są dostępne
+Name[pt]=Estão Disponíveis Actualizações
+Name[pt_BR]=Atualizações estão disponíveis
+Name[sk]=Sú dostupné aktualizácie
+Name[sl]=Na voljo so posodobitve
+Name[sr]=Доступне допуне
+Name[sr@ijekavian]=Доступне допуне
+Name[sr@ijekavianlatin]=Dostupne dopune
+Name[sr@latin]=Dostupne dopune
+Name[sv]=Uppdateringar är tillgängliga
+Name[uk]=Доступні оновлення
+Name[x-test]=xxUpdates Are Availablexx
+Name[zh_CN]=更新可用
+Comment=Updates Available
+Comment[ca]=Actualitzacions disponibles
+Comment[ca@valencia]=Actualitzacions disponibles
+Comment[da]=Opdateringer tilgængelige
+Comment[de]=Aktualisierungen verfügbar
+Comment[el]=Διαθέσιμες ενημερώσεις
+Comment[en_GB]=Updates Available
+Comment[es]=Actualizaciones disponibles
+Comment[fi]=Päivityksiä saatavilla
+Comment[fr]=Mises à jour disponibles
+Comment[gl]=Actualizacións dispoñíbeis
+Comment[it]=Aggiornamenti disponibili
+Comment[ko]=업데이트 사용 가능
+Comment[nb]=Oppgraderinger tilgjengelige
+Comment[nl]=Bijwerken is beschikbaar
+Comment[nn]=Oppdateringar tilgjengelege
+Comment[pl]=Dostępne uaktualnienia
+Comment[pt]=Actualizações Disponíveis
+Comment[pt_BR]=Atualizações disponíveis
+Comment[sk]=Dostupné aktualizácie
+Comment[sl]=Na voljo so posodobitve
+Comment[sr]=Доступне су нове допуне
+Comment[sr@ijekavian]=Доступне су нове допуне
+Comment[sr@ijekavianlatin]=Dostupne su nove dopune
+Comment[sr@latin]=Dostupne su nove dopune
+Comment[sv]=Uppdateringar tillgängliga
+Comment[uk]=Доступні оновлення
+Comment[x-test]=xxUpdates Availablexx
+Comment[zh_CN]=更新可用
+Action=Popup
+
+

--- a/libdiscover/resources/muonabstractnotifier.notifyrc
+++ /dev/null
@@ -1,94 +1,1 @@
-[Global]
-IconName=applications-system
-Comment=Muon
-Comment[ca]=Muon
-Comment[ca@valencia]=Muon
-Comment[da]=Muon
-Comment[de]=Muon
-Comment[el]=Muon
-Comment[en_GB]=Muon
-Comment[es]=Muon
-Comment[fi]=Muon
-Comment[fr]=Muon
-Comment[gl]=Muon
-Comment[it]=Muon
-Comment[ko]=Muon
-Comment[nb]=Muon
-Comment[nl]=Muon
-Comment[nn]=Muon
-Comment[pl]=Muon
-Comment[pt]=Muon
-Comment[pt_BR]=Muon
-Comment[sk]=Muon
-Comment[sl]=Muon
-Comment[sr]=Муон
-Comment[sr@ijekavian]=Муон
-Comment[sr@ijekavianlatin]=Muon
-Comment[sr@latin]=Muon
-Comment[sv]=Muon
-Comment[uk]=Muon
-Comment[x-test]=xxMuonxx
-Comment[zh_CN]=Muon
 
-[Event/Update]
-Name=Updates Are Available
-Name[ca]=Hi ha actualitzacions disponibles
-Name[ca@valencia]=Hi ha actualitzacions disponibles
-Name[da]=Opdateringer tilgængelige
-Name[de]=Es sind Aktualisierungen verfügbar
-Name[el]=Υπάρχουν διαθέσιμες ενημερώσεις
-Name[en_GB]=Updates Are Available
-Name[es]=Existen actualizaciones disponibles
-Name[fi]=Päivityksiä on saatavilla
-Name[fr]=Des mises à jour sont disponibles
-Name[gl]=Hai actualizacións dispoñíbeis
-Name[it]=Sono disponibili aggiornamenti
-Name[ko]=업데이트 사용 가능
-Name[nb]=Oppgraderinger er tilgjengelige
-Name[nl]=Bijwerken is beschikbaar
-Name[nn]=Oppdateringar er tilgjengelege
-Name[pl]=Uaktualnienia są dostępne
-Name[pt]=Estão Disponíveis Actualizações
-Name[pt_BR]=Atualizações estão disponíveis
-Name[sk]=Sú dostupné aktualizácie
-Name[sl]=Na voljo so posodobitve
-Name[sr]=Доступне допуне
-Name[sr@ijekavian]=Доступне допуне
-Name[sr@ijekavianlatin]=Dostupne dopune
-Name[sr@latin]=Dostupne dopune
-Name[sv]=Uppdateringar är tillgängliga
-Name[uk]=Доступні оновлення
-Name[x-test]=xxUpdates Are Availablexx
-Name[zh_CN]=更新可用
-Comment=Updates Available
-Comment[ca]=Actualitzacions disponibles
-Comment[ca@valencia]=Actualitzacions disponibles
-Comment[da]=Opdateringer tilgængelige
-Comment[de]=Aktualisierungen verfügbar
-Comment[el]=Διαθέσιμες ενημερώσεις
-Comment[en_GB]=Updates Available
-Comment[es]=Actualizaciones disponibles
-Comment[fi]=Päivityksiä saatavilla
-Comment[fr]=Mises à jour disponibles
-Comment[gl]=Actualizacións dispoñíbeis
-Comment[it]=Aggiornamenti disponibili
-Comment[ko]=업데이트 사용 가능
-Comment[nb]=Oppgraderinger tilgjengelige
-Comment[nl]=Bijwerken is beschikbaar
-Comment[nn]=Oppdateringar tilgjengelege
-Comment[pl]=Dostępne uaktualnienia
-Comment[pt]=Actualizações Disponíveis
-Comment[pt_BR]=Atualizações disponíveis
-Comment[sk]=Dostupné aktualizácie
-Comment[sl]=Na voljo so posodobitve
-Comment[sr]=Доступне су нове допуне
-Comment[sr@ijekavian]=Доступне су нове допуне
-Comment[sr@ijekavianlatin]=Dostupne su nove dopune
-Comment[sr@latin]=Dostupne su nove dopune
-Comment[sv]=Uppdateringar tillgängliga
-Comment[uk]=Доступні оновлення
-Comment[x-test]=xxUpdates Availablexx
-Comment[zh_CN]=更新可用
-Action=Popup
-
-

--- a/notifier/CMakeLists.txt
+++ b/notifier/CMakeLists.txt
@@ -1,4 +1,4 @@
-add_definitions(-DTRANSLATION_DOMAIN=\"muon-notifier\")
+add_definitions(-DTRANSLATION_DOMAIN=\"plasma-discover-notifier\")
 
 add_library(DiscoverNotifierDeclarativePlugin BackendNotifierFactory.cpp DiscoverNotifier.cpp DiscoverNotifierDeclarativePlugin.cpp)
 target_link_libraries(DiscoverNotifierDeclarativePlugin

--- a/notifier/DiscoverNotifier.cpp
+++ b/notifier/DiscoverNotifier.cpp
@@ -51,7 +51,7 @@
 
 void DiscoverNotifier::configurationChanged()
 {
-    KConfig notifierConfig(QStringLiteral("muon-notifierrc"), KConfig::NoGlobals);
+    KConfig notifierConfig(QStringLiteral("plasma-discover-notifierrc"), KConfig::NoGlobals);
 
     KConfigGroup notifyTypeGroup(&notifierConfig, "NotificationType");
     m_verbose = notifyTypeGroup.readEntry("Verbose", false);
@@ -59,7 +59,7 @@
 
 void DiscoverNotifier::showMuon()
 {
-    KRun::runCommand(QStringLiteral("muon-discover --mode update"), nullptr);
+    KRun::runCommand(QStringLiteral("plasma-discover --mode update"), nullptr);
 }
 
 bool DiscoverNotifier::isSystemUpToDate() const

--- a/updater/CMakeLists.txt
+++ b/updater/CMakeLists.txt
@@ -1,6 +1,6 @@
 include_directories(../libdiscover)
 
-set(muon_updater_SRCS
+set(plasmadiscover_updater_SRCS
     main.cpp
     MainWindow.cpp
     ChangelogWidget.cpp
@@ -9,14 +9,14 @@
     KActionMessageWidget.cpp
     UpdateDelegate.cpp)
 
-ki18n_wrap_ui(muon_updater_SRCS ProgressWidget.ui
+ki18n_wrap_ui(plasmadiscover_updater_SRCS ProgressWidget.ui
                                     UpdaterCentralWidget.ui
                                     UpdaterWidgetNoUpdates.ui
 )
 
-add_executable(muon-updater ${muon_updater_SRCS})
+add_executable(plasma-discover-updater ${plasmadiscover_updater_SRCS})
 
-target_link_libraries(muon-updater KF5::CoreAddons
+target_link_libraries(plasma-discover-updater KF5::CoreAddons
                                    KF5::KIOWidgets
                                    KF5::Solid
                                    KF5::I18n
@@ -26,14 +26,14 @@
                                    Discover::Common)
 
 if (ENABLE_QZEITGEIST)
-    target_link_libraries(muon-updater ${QZEITGEIST_LIBRARY})
+    target_link_libraries(plasma-discover-updater ${QZEITGEIST_LIBRARY})
 endif ()
 
-install(TARGETS muon-updater ${INSTALL_TARGETS_DEFAULT_ARGS} )
+install(TARGETS plasma-discover-updater ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
 ########### install files ###############
 
-install( FILES muonupdaterui.rc  DESTINATION  ${KXMLGUI_INSTALL_DIR}/muonupdater )
-install( PROGRAMS muon-updater.desktop  DESTINATION ${XDG_APPS_INSTALL_DIR} )
+install( FILES plasmadiscoverupdaterui.rc  DESTINATION  ${KXMLGUI_INSTALL_DIR}/plasmadiscoverupdater )
+install( PROGRAMS plasma-discover-updater.desktop  DESTINATION ${XDG_APPS_INSTALL_DIR} )
 
 

--- a/updater/ChangelogWidget.cpp
+++ b/updater/ChangelogWidget.cpp
@@ -173,9 +173,9 @@
 void ChangelogWidget::showMore(const QUrl& package)
 {
     if (package.scheme() == QLatin1String("package")) {
-        bool b = QProcess::startDetached(QStringLiteral("muon-discover"), { QStringLiteral("--application"), package.path() });
+        bool b = QProcess::startDetached(QStringLiteral("plasma-discover"), { QStringLiteral("--application"), package.path() });
         if (!b) {
-            qWarning() << "Couldn't launch muon-discover";
+            qWarning() << "Couldn't launch plasma-discover";
         }
     } else {
         QDesktopServices::openUrl(package);

--- a/updater/main.cpp
+++ b/updater/main.cpp
@@ -33,8 +33,8 @@
 {
     QApplication app(argc, argv);
     app.setWindowIcon(QIcon::fromTheme(QStringLiteral("system-software-update")));
-    KLocalizedString::setApplicationDomain("muon-updater");
-    KAboutData about(QStringLiteral("muonupdater"), i18n("Update Manager"), version, i18n("An update manager"),
+    KLocalizedString::setApplicationDomain("plasma-discover-updater");
+    KAboutData about(QStringLiteral("plasmadiscoverupdater"), i18n("Update Manager"), version, i18n("An update manager"),
                      KAboutLicense::GPL, i18n("©2010-2013 Jonathan Thomas"), QString());
     about.addAuthor(i18n("Jonathan Thomas"), QString(), QStringLiteral("echidnaman@kubuntu.org"));
     about.addAuthor(i18n("Aleix Pol"), QString(), QStringLiteral("aleixpol@kde.org"));

--- a/updater/muon-updater.desktop
+++ /dev/null
@@ -1,60 +1,1 @@
-[Desktop Entry]
-Name=Update Manager
-Name[ca]=Gestor d'actualitzacions
-Name[ca@valencia]=Gestor d'actualitzacions
-Name[da]=Håndtering af opdateringer
-Name[de]=Aktualisierungsverwaltung
-Name[en_GB]=Update Manager
-Name[es]=Gestor de actualizaciones
-Name[fi]=Päivitystenhallinta
-Name[gl]=Xestor de actualizacións
-Name[nl]=Beheerder voor bijwerken
-Name[nn]=Oppdateringshandsamar
-Name[pl]=Zarządzanie uaktualnieniami
-Name[pt]=Gestor de Actualizações
-Name[pt_BR]=Gerenciador de atualizações
-Name[sk]=Správca aktualizácií
-Name[sl]=Upravljalnik posodobitev
-Name[sr]=Менаџер ажурирања
-Name[sr@ijekavian]=Менаџер ажурирања
-Name[sr@ijekavianlatin]=Menadžer ažuriranja
-Name[sr@latin]=Menadžer ažuriranja
-Name[sv]=Uppdateringshanterare
-Name[uk]=Керування оновленнями
-Name[x-test]=xxUpdate Managerxx
-Name[zh_CN]=更新管理器
-Exec=muon-updater
-Icon=system-software-update
-Type=Application
-X-DocPath=muon/index.html
-GenericName=Update Manager
-GenericName[ca]=Gestor d'actualitzacions
-GenericName[ca@valencia]=Gestor d'actualitzacions
-GenericName[da]=Håndtering af opdateringer
-GenericName[de]=Aktualisierungsverwaltung
-GenericName[el]=Διαχειριστής ενημερώσεων
-GenericName[en_GB]=Update Manager
-GenericName[es]=Gestor de actualizaciones
-GenericName[fi]=Päivitystenhallinta
-GenericName[fr]=Gestionnaire de mises à jour
-GenericName[gl]=Xestor de actualizacións
-GenericName[it]=Gestore degli aggiornamenti
-GenericName[ko]=업데이트 관리자
-GenericName[nb]=Oppdateringsbehandler
-GenericName[nl]=Beheerder voor bijwerken
-GenericName[nn]=Oppdateringshandsamar
-GenericName[pl]=Zarządzanie uaktualnieniami
-GenericName[pt]=Gestor de Actualizações
-GenericName[pt_BR]=Gerenciador de atualizações
-GenericName[sk]=Správca aktualizácií
-GenericName[sl]=Upravljalnik posodobitev
-GenericName[sr]=Менаџер ажурирања
-GenericName[sr@ijekavian]=Менаџер ажурирања
-GenericName[sr@ijekavianlatin]=Menadžer ažuriranja
-GenericName[sr@latin]=Menadžer ažuriranja
-GenericName[sv]=Uppdateringshanterare
-GenericName[uk]=Керування оновленнями
-GenericName[x-test]=xxUpdate Managerxx
-GenericName[zh_CN]=更新管理器
-Categories=Qt;KDE;System;
 

--- a/updater/muonupdaterui.rc
+++ /dev/null
@@ -1,40 +1,1 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<gui name="muon-updater"
-     version="1"
-     xmlns="http://www.kde.org/standards/kxmlgui/1.0"
-     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-     xsi:schemaLocation="http://www.kde.org/standards/kxmlgui/1.0
-                         http://www.kde.org/standards/kxmlgui/1.0/kxmlgui.xsd">
-    <MenuBar>
-        <Menu name="file" >
-            <Separator/>
-            <Action name="save_download_list" />
-            <Action name="download_from_list" />
-            <Action name="load_archives" />
-            <Separator/>
-            <Action name="file_quit" />
-        </Menu>
-        <Menu name="edit" >
-            <Action name="undo" />
-            <Action name="redo" />
-            <Separator/>
-            <Action name="update" />
-            <Separator/>
-            <Action name="apply" />
-        </Menu>
-        <Menu name="view" >
-            <Action name="history" />
-        </Menu>
-        <Menu name="settings">
-          <Separator />
-          <Action name="software_properties" />
-        </Menu>
-    </MenuBar>
-    <ToolBar name="mainToolBar">
-        <Action name="apply" />
-        <Action name="file_quit" />
-        <Separator/>
-        <Action name="update" />
-    </ToolBar>
-</gui>
 

--- /dev/null
+++ b/updater/plasma-discover-updater.desktop
@@ -1,1 +1,60 @@
+[Desktop Entry]
+Name=Update Manager
+Name[ca]=Gestor d'actualitzacions
+Name[ca@valencia]=Gestor d'actualitzacions
+Name[da]=Håndtering af opdateringer
+Name[de]=Aktualisierungsverwaltung
+Name[en_GB]=Update Manager
+Name[es]=Gestor de actualizaciones
+Name[fi]=Päivitystenhallinta
+Name[gl]=Xestor de actualizacións
+Name[nl]=Beheerder voor bijwerken
+Name[nn]=Oppdateringshandsamar
+Name[pl]=Zarządzanie uaktualnieniami
+Name[pt]=Gestor de Actualizações
+Name[pt_BR]=Gerenciador de atualizações
+Name[sk]=Správca aktualizácií
+Name[sl]=Upravljalnik posodobitev
+Name[sr]=Менаџер ажурирања
+Name[sr@ijekavian]=Менаџер ажурирања
+Name[sr@ijekavianlatin]=Menadžer ažuriranja
+Name[sr@latin]=Menadžer ažuriranja
+Name[sv]=Uppdateringshanterare
+Name[uk]=Керування оновленнями
+Name[x-test]=xxUpdate Managerxx
+Name[zh_CN]=更新管理器
+Exec=plasma-discover-updater
+Icon=system-software-update
+Type=Application
+X-DocPath=muon/index.html
+GenericName=Update Manager
+GenericName[ca]=Gestor d'actualitzacions
+GenericName[ca@valencia]=Gestor d'actualitzacions
+GenericName[da]=Håndtering af opdateringer
+GenericName[de]=Aktualisierungsverwaltung
+GenericName[el]=Διαχειριστής ενημερώσεων
+GenericName[en_GB]=Update Manager
+GenericName[es]=Gestor de actualizaciones
+GenericName[fi]=Päivitystenhallinta
+GenericName[fr]=Gestionnaire de mises à jour
+GenericName[gl]=Xestor de actualizacións
+GenericName[it]=Gestore degli aggiornamenti
+GenericName[ko]=업데이트 관리자
+GenericName[nb]=Oppdateringsbehandler
+GenericName[nl]=Beheerder voor bijwerken
+GenericName[nn]=Oppdateringshandsamar
+GenericName[pl]=Zarządzanie uaktualnieniami
+GenericName[pt]=Gestor de Actualizações
+GenericName[pt_BR]=Gerenciador de atualizações
+GenericName[sk]=Správca aktualizácií
+GenericName[sl]=Upravljalnik posodobitev
+GenericName[sr]=Менаџер ажурирања
+GenericName[sr@ijekavian]=Менаџер ажурирања
+GenericName[sr@ijekavianlatin]=Menadžer ažuriranja
+GenericName[sr@latin]=Menadžer ažuriranja
+GenericName[sv]=Uppdateringshanterare
+GenericName[uk]=Керування оновленнями
+GenericName[x-test]=xxUpdate Managerxx
+GenericName[zh_CN]=更新管理器
+Categories=Qt;KDE;System;
 

--- /dev/null
+++ b/updater/plasmadiscoverupdaterui.rc
@@ -1,1 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<gui name="plasma-discover-updater"
+     version="1"
+     xmlns="http://www.kde.org/standards/kxmlgui/1.0"
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://www.kde.org/standards/kxmlgui/1.0
+                         http://www.kde.org/standards/kxmlgui/1.0/kxmlgui.xsd">
+    <MenuBar>
+        <Menu name="file" >
+            <Separator/>
+            <Action name="save_download_list" />
+            <Action name="download_from_list" />
+            <Action name="load_archives" />
+            <Separator/>
+            <Action name="file_quit" />
+        </Menu>
+        <Menu name="edit" >
+            <Action name="undo" />
+            <Action name="redo" />
+            <Separator/>
+            <Action name="update" />
+            <Separator/>
+            <Action name="apply" />
+        </Menu>
+        <Menu name="view" >
+            <Action name="history" />
+        </Menu>
+        <Menu name="settings">
+          <Separator />
+          <Action name="software_properties" />
+        </Menu>
+    </MenuBar>
+    <ToolBar name="mainToolBar">
+        <Action name="apply" />
+        <Action name="file_quit" />
+        <Separator/>
+        <Action name="update" />
+    </ToolBar>
+</gui>
 

